### PR TITLE
Switch to CustomRouteController from library-go

### PR DESCRIPTION
**Pull request by slaskawi**
https://issues.redhat.com/browse/AUTH-14

This is a proof for https://github.com/openshift/library-go/pull/1138

### CODE REVIEW

**Comment by slaskawi** [comment]
On my box, the `TestCustomRouterCerts` test is fine. Let's give them a few more tries...

/retest

File: pkg/operator/starter.go [hunk 1]
```diff
@@ -9,6 +9,8 @@ import (
 	"os"
 	"time"
 
+	"github.com/openshift/library-go/pkg/operator/customroute"
+	"github.com/openshift/library-go/pkg/operator/resource/resourceread"
```
Comment (stlaz) [inline pkg/operator/starter.go:13]: order

**Review by stlaz — CHANGES_REQUESTED** [review]

File: pkg/operator/starter.go (hunk 1 above)
Comment (slaskawi) [reply pkg/operator/starter.go]: Fixed.

**Pull request closed without merging**